mrp_finished_backorder_product/models/mrp_production.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import api, fields, models
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+    finished_move_ids_from_backorders = fields.One2many(
+        "stock.move.line",
+        compute="_compute_finished_backorders",
```

Comment:
the Many2one relation from the stock.move.line to the manufacturing order is mandatory, isn't it?

Reply:
I think that its mandatory